### ISSUE TITLE
Remove default youtube video

### ISSUE THREAD

**Issue by up2code** [sarbagyastha/youtube_player_flutter#93]
When I run video id that error (In my case is error code 150). Sometime it fallback to video https://www.youtube.com/watch?v=50kklGefAcs and I found that this video id had defined in lib/src/player.dart

Should it be removed?

Example video error
- https://www.youtube.com/watch?v=yilfICzwrSM
- https://www.youtube.com/watch?v=-j9YNUsBgDE

Test base on
- youtube_player_flutter : 5.0.0+1
- Flutter : 1.9.1+hotfix.2
- Physical android device (Android 9)

**Comment by sarbagyastha** [comment]
@up2code I forgot to remove it. Thank you! Will remove it.

**Comment by SayaGames** [comment]
Yes, I'm going to report default movie issue. 

VideoData {"duration":608.401,"videoUrl":"https://www.youtube.com/watch?v=yhrLHNFF8Ig","availableQualityLevels":["hd1080","hd720","large","medium","small","tiny","auto"],"videoEmbedCode":"<iframe width=\"100\" height=\"100\" src=\"https://www.youtube.com/embed/50kklGefAcs\" frameborder=\"0\" allow=\"accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture\" allowfullscreen></iframe>"}

New Nepali Movie - 2017/2074 | Bhanchhu Aaja || Ma Yesto Geet Gaauchu || Ft Pooja Sharma, Paul Shah

Also I get Error code 400 No connection when the player starts play and 1-2 seconds later.

**Comment by sarbagyastha** [comment]
@SayaGames are you sure internet was working correctly? Can you provide with code snippet to recreate the issue ?

**Comment by SayaGames** [comment]
I tried at home wifi, at work wifi and mobile network. (real device and android emulator)

* First: Player is shown correctly, (when page is loaded)
![image](https://user-images.githubusercontent.com/31693836/67261828-fc898780-f4aa-11e9-8f31-4a8ea80cf5d1.png)

* but 2-3 seconds later, player status is changing to error display
![image](https://user-images.githubusercontent.com/31693836/67261728-9dc40e00-f4aa-11e9-9ce8-9ad37ba68acd.png)

* It's weird that, if I back to previous page and re-enter the player page again, everything is normal. 
* Normal is: Error display is gone. And I also mentioned in the issue #94, intermittent playback is (sometimes) gone. Playing correctly. 

 * My Player code is below: 
                  Stack(
                    children: <Widget>[
                      Center(
                        child: YoutubePlayer(
                          context: context,
                          videoId: url1,
                          inFullScreen: false,
                          flags: YoutubePlayerFlags(
                            mute: false,
                            autoPlay: false,
                            loop:  false,
                            forceHideAnnotation: true,
                            showVideoProgressIndicator: true,
                            disableDragSeek: false,
                            hideThumbnail: false,
                            isLive: false,
                          ),
                          progressIndicatorColor: Color(0xFFFF0000),
                          topActions: <Widget>[
                            Text(globals.title,
                              style: TextStyle(
                                  color: Colors.white,
                                  fontSize: 13,
                                  fontWeight: FontWeight.normal
                              ),
                            ),
                          ],
                          bottomActions: [
                            CurrentPosition(),
                            ProgressBar(isExpanded: true),
                            TotalDuration(),
                          ],
                          onPlayerInitialized: (controller) {
                            _controller1 = controller;
                          },
                        ),
                      ),

                    ],
                  ),

**Comment by Xoshbin** [comment]
It looks like it's same behavior as mine https://github.com/sarbagyastha/youtube_player_flutter/issues/79#issuecomment-542138560

**Comment by SayaGames** [comment]
Hi, 
Newly encountered for me: I also saw the video instead of black error display now: 
New Nepali Movie - 2017/2074 | Bhanchhu Aaja || Ma Yesto Geet Gaauchu || Ft Pooja Sharma, Paul Shah

**Comment by sarbagyastha** [comment]
Does the issue persist on Version 5.1.0 ?

**Comment by up2code** [comment]
@sarbagyastha On version 5.1.0 fallback video not show anymore. Thank you very much! but now for video error sometime it display only play button or circle progress indicator instead of error message. 

I had tested by create playlist screen with two ListTile (2 youtube url with error) and tap to change videoId value.  But that minor issue for me. So maybe we can close this?

**Comment by sarbagyastha** [comment]
@up2code With version 5.2.0, there's slight change in API. And it should fix your problem.

**Comment by up2code** [comment]
@sarbagyastha Thank you so much for your hard work! It show error now when tap on video with error. But when I tap back to load playable url, player not update and still display previous error message and below is exception log that throw in console.

```
E/flutter ( 3207): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method evaluateJavascript on channel plugins.flutter.io/webview_0)
E/flutter ( 3207): #0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:314:7)
E/flutter ( 3207): <asynchronous suspension>
E/flutter ( 3207): #1      MethodChannelWebViewPlatform.evaluateJavascript (package:webview_flutter/src/webview_method_channel.dart:90:21)
E/flutter ( 3207): #2      WebViewController.evaluateJavascript (package:webview_flutter/webview_flutter.dart:626:39)
E/flutter ( 3207): #3      YoutubePlayerController._evaluateJS (package:youtube_player_flutter/src/utils/youtube_player_controller.dart:155:30)
E/flutter ( 3207): #4      YoutubePlayerController.load (package:youtube_player_flutter/src/utils/youtube_player_controller.dart:170:5)
E/flutter ( 3207): #5      _YoutubePlaylistPageState.buildPlayer.<anonymous closure> (package:vocadb/pages/youtube_playlist/youtube_playlist_page.dart:130:29)

```

Test on android 9 physical device with version 5.2.0

**Comment by sarbagyastha** [comment]
Could you please attach code snippet, how you're loading videos?

**Comment by up2code** [comment]
@sarbagyastha Sure, below I test with simple stateful widget with 2 valid and 1 error video. No problem when try tap ListTile to load valid videoId. But if I tap on error one error message will display on player (But no exception yet) and when I try to tap on valid one, Exception message will show in console. Did I used api correctly?
```
class _TestWebViewState extends State<TestWebView> {
  YoutubePlayerController _controller;

  @override
  void dispose() {
    _controller.dispose();
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Test playlist'),
      ),
      body: Column(
        children: <Widget>[
          YoutubePlayer(
            context: context,
            initialVideoId: 'AufydOsiD6M',
            flags: YoutubePlayerFlags(
              autoPlay: false,
              showVideoProgressIndicator: true,
            ),
            onPlayerInitialized: (controller) {
              _controller = controller;
            },
          ),
          Expanded(
            child: ListView(
              children: <Widget>[
                ListTile(
                  title: Text('Valid 1'),
                  onTap: () {
                    _controller.load('AufydOsiD6M');
                  },
                ),
                ListTile(
                  title: Text('Valid 2'),
                  onTap: () {
                    _controller.load('V-189K9CcQw');
                  },
                ),
                ListTile(
                  title: Text('Error video'),
                  onTap: () {
                    _controller.load('j9YNUsBgDE');
                  },
                ),
              ],
            ),
          )
        ],
      ),
    );
  }
}
```

**Comment by sarbagyastha** [comment]
I guess it should be fixed with 5.2.0+1, have you tried with that ?

**Comment by up2code** [comment]
@sarbagyastha I just tried with 5.2.0+1. Now valid video can play after previous error video, but error message won't show up if next one I tap on error video. I had tested and found it will display error message when I play error video first. If I tap valid one and tap to error video, It show only play button.

You can try on my code snippet on above one.

**Comment by sarbagyastha** [comment]
Oh! I think a reload action should be added for such scenario, will look at it once i get free.

**Comment by sarbagyastha** [comment]
Should fix with [Version 6.0.0](https://pub.dev/packages/youtube_player_flutter#600).

**Comment by up2code** [comment]
Seem like it fixed now. Thank you so much!
